keel-core/src/main/kotlin/com/netflix/spinnaker/keel/api/DeliveryArtifact.kt

```diff
@@ -1,12 +1,26 @@
 package com.netflix.spinnaker.keel.api
 
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonValue
+
 data class DeliveryArtifact(
   val name: String,
   val type: ArtifactType = ArtifactType.DEB
 )
 
-enum class ArtifactType {
-  DEB
+enum class ArtifactType(@JsonValue val friendlyName: String) {
+  DEB("deb"),
+  DOCKER_IMAGE("docker");
```

Comment:
This will conflict with @emjburns' change in #556. I'll wait for her to merge and then resolve the conflict.